app/views/users/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
```

Comment:
I may have changed my mind. If we give them the username, they now have the email address as well as the username. Let's either keep it as email or remove it entirely with a message like "Welcome! Someone signed up for ... with this email address. If it wasn't you, just ignore it."